app/components/appeals-app-services/variables.tf

```diff
@@ -111,6 +116,12 @@ variable "logger_level" {
   default     = "info"
 }
 
+variable "monitoring_alerts_enabled" {
+  default     = false
+  description = "Indicates whether Azure Monitor alerts are enabled for App Service"
```

Comment:
Small nitpicky thing, but since I've added a module for the function apps too, maybe we should drop the "for App service" bit.